pkg/types/binary.go

```diff
@@ -52,7 +53,7 @@ func (binary *Binary) UnmarshalText(text []byte) error {
 	b := make([]byte, hex.DecodedLen(len(text)))
 	_, err := hex.Decode(b, text)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "can't decode hex %q into %T", text, b)
```

Comment:
`%T` of `b` will always be `[]byte`, so I think there's little value in adding this to the message.